src/components/Header/index.tsx

```diff
@@ -320,7 +320,10 @@ export default function Header() {
           <StyledNavLink id={`vote-nav-link`} to={'/vote'}>
             {t('header.vote')}
           </StyledNavLink>
-          <StyledExternalLink id={`charts-nav-link`} href={ANALYTICS_PAGE}>
+          <StyledNavLink id={`stake-nav-link`} to={'/IDO'}>
```

Comment:
We are already out of space on the headers section. Can we move this to the dropdown section (with About, Tutorials, etc.) instead?

Reply:
I have removed the IDO page from the main header and moved it to the drop down menu. Chose a calendar icon (we may want something else). Also added the MenuNavItem to navigate to a page instead of popping a new window.